### PR TITLE
fix: electron `windowTracker` crash when reloading and killing scripts

### PR DESCRIPTION
**Pull request by smolgumball**
## resolves crash in `electron/windowTracker.js` during reload

* currently crashes when running `Reloads -> Reload & Kill All Scripts` menu command
* this fix adds extra checks to window position tracking code to ensure Electron window handles are valid before attempting to access their properties

**Before fix**
image

**After fix:**
l6fet9gVOI

### CODE REVIEW

**Review by phyzical — APPROVED** [review]
LGTM, will approve when gif showing what the new result looks like.

**Comment by smolgumball** [comment]
@phyzical New result would be a blank screenshot of my desktop :) Did you still need that?

**Comment by phyzical** [comment]
ah i assumed it would resolve the issue an reload the scripts safely, or it just more about handling the issue more gracefully? 

if so yeah as silly as it sounds it will atleast show you've tested it properly :) (not that i dont trust you havnt :P )

**Comment by smolgumball** [comment]
@phyzical Added screenshot to original issue

You mentioned "resolve the issue [and] reload the scripts safely", but I believe the desired outcome from the `Reload & Kill All Scripts` command is that the game reboots with no running scripts. This was the behavior previously, and still is. 

This fix is solely focused on addressing a regression due to dc2f00057aa298e30cd864db65ace15f7fad3eb0

**Comment by phyzical** [comment]
thanks for the picture, i actually was getting this the other day on the dev branch. buuuut i think it actually has since been fixed.

is there any chance you can try to replicate it on the latest dev? (make sure you rebase dev properly ect :))

**Comment by smolgumball** [comment]
@phyzical Still happening on latest `dev`

Current:
image

After merging 3431997315e0e3591409ac66c06a7be289f027d2:
image

**Comment by phyzical** [comment]
coolies thx

**Pull request merged**
